src/markdown.c

```diff
@@ -1517,6 +1536,48 @@ parse_blockquote(struct buf *ob, struct sd_markdown *rndr, uint8_t *data, size_t
 	return end;
 }
 
+/* parse_blockspoiler • handles parsing of a blockspoiler fragment */
+static size_t
+parse_blockspoiler(struct buf *ob, struct sd_markdown *rndr, uint8_t *data, size_t size)
+{
+	size_t beg, end = 0, pre, work_size = 0;
+	uint8_t *work_data = 0;
+	struct buf *out = 0;
+
+	out = rndr_newbuf(rndr, BUFFER_BLOCK);
+	beg = 0;
+	while (beg < size) {
+		for (end = beg + 1; end < size && data[end - 1] != '\n'; end++);
+
+		pre = prefix_blockspoiler(data + beg, end - beg);
+
+		if (pre)
+			beg += pre; /* skipping prefix */
+
+		/* empty line followed by non-blockspoiler line */
+		else if (is_empty(data + beg, end - beg) &&
```

Comment:
🔕 So is this saying that if we see something like:

```
>!
Testing
```

It will ignore the blockspoiler entirely? I think that's probably pretty fine (as that's weird behavior), just trying to work it out since this doesn't check the value of `pre` at all.